libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.html

```diff
@@ -34,11 +36,13 @@
   </button>
   <span
     ><label
-      *ngIf="hasLabelComponent"
+      *ngIf="hasLabelComponent || labelText"
       [for]="toggle.id"
       [attr.id]="labelId"
       class="sky-toggle-switch-label"
-      ><ng-container *ngTemplateOutlet="labelContent"></ng-container></label
+      ><ng-container *ngIf="labelText && !labelHidden; else labelContent">
```

Comment:
Won't this show the "legacy" `sky-toggle-switch-label` component if `labelText` exists, but `labelHidden` is set to true? Should it?
```
<sky-toggle-switch labelText="My custom text" [labelHidden]="true">
  <sky-toggle-switch-label> legacy label </sky-toggle-switch-label>
</sky-toggle-switch>
```